add option to disable the landing page (#801)
apollographql/router: CHANGELOG.md

 # [v0.1.0-preview.3] (unreleased) - 2022-mm-dd
## ❗ BREAKING ❗
## 🚀 Features

- **Add an option to disable the landing page** ([PR #801](https://github.com/apollographql/router/pull/801))

>   While displaying a landing page could be useful in development, we might not want to display it,
>   so the router can be configured to not display any landing page:

Reviewer: I always like making clear what the default is:
```suggestion
  By default the router will display a landing page, which could be useful in development. If this is not
  desirable the router can be configured to not display this landing page:
```
